parlai/agents/transformer/modules.py

```diff
@@ -334,17 +401,23 @@ def __init__(
         attention_dropout=0.0,
         relu_dropout=0.0,
         dropout=0.0,
+        activation='relu',
+        variant=None,
     ):
         super().__init__()
         self.dim = embedding_size
         self.ffn_dim = ffn_size
+        self.activation = activation
+        self.variant = variant
         self.attention = MultiHeadAttention(
             n_heads, embedding_size,
             dropout=attention_dropout,  # --attention-dropout
         )
-        self.norm1 = nn.LayerNorm(embedding_size)
-        self.ffn = TransformerFFN(embedding_size, ffn_size, relu_dropout=relu_dropout)
-        self.norm2 = nn.LayerNorm(embedding_size)
+        self.norm1 = nn.LayerNorm(embedding_size, eps=LAYER_NORM_EPS)
```

Comment:
why did you change the epsilon here?

Reply:
because it was specified in some places and not in others. I figure it's better to use a consistent value everywhere.